scripts/sysman/device.yml

```diff
@@ -100,14 +117,71 @@ members:
       desc: "[out] Indicates if the device has been repaired"
 --- #--------------------------------------------------------------------------
 type: struct
+desc: "Device reset properties"
+class: $sDevice
+name: $s_reset_properties_t
+base: $s_base_properties_t
+members:
+    - type: ze_bool_t
+      name: force
+      desc: |
+            [in] If set to true, all applications that are currently using the device will be forcibly killed.
+    - type: $s_reset_type_t
+      name: resetType
+      desc: "[in] Type of reset needs to be performed"
+--- #--------------------------------------------------------------------------
+type: struct
+desc: "Device universal unique id (UUID)"
+version: "1.7"
+class: $sDevice
+name: $s_uuid_t
+members:
+   - type: uint8_t
+     name: "id[$S_MAX_UUID_SIZE]"
+     desc: "[out] opaque data representing a device UUID"
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Supported device types"
+version: "1.7"
+class: $sDevice
+name: $s_device_type_t
+etors:
+    - name: GPU
+      value: "1"
+      desc: "Graphics Processing Unit"
+    - name: CPU
+      desc: "Central Processing Unit"
+    - name: FPGA
+      desc: "Field Programmable Gate Array"
+    - name: MCA
+      desc: "Memory Copy Accelerator"
+    - name: VPU
+      desc: "Vision Processing Unit"
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "Supported device property flags"
+version: "1.7"
+class: $sDevice
+name: $s_device_property_flags_t
+etors:
+    - name: INTEGRATED
+      desc: "Device is integrated with the Host."
+    - name: SUBDEVICE
+      desc: "Device handle used for query represents a sub-device."
+    - name: ECC
+      desc: "Device supports error correction memory access."
+    - name: ONDEMANDPAGING
+      desc: "Device supports on-demand page-faulting."
+--- #--------------------------------------------------------------------------
+type: struct
 desc: "Device properties"
 class: $sDevice
 name: $s_device_properties_t
 base: $s_base_properties_t
 members:
     - type: $x_device_properties_t
       name: core
-      desc: "[out] Core device properties"
+      desc: "[out] (Deprecated, use $s_uuid_t in the extended structure) Core device properties"
```

Comment:
this requires also versioning. We need to keep the previous line for 1.0 up to 1.6 and then the new line for 1.7.

Reply:
Fixed